bin/uglifyjs

```diff
@@ -212,12 +212,26 @@ if (ARGS.quotes === true) {
     ARGS.quotes = 3;
 }
 
+if (ARGS.mangle_props === true) {
+    ARGS.mangle_props = 1;
+}
+
+if (ARGS.mangle_props === "unquoted") {
+    ARGS.mangle_props = 2;
+}
+
 var OUTPUT_OPTIONS = {
-    beautify    : BEAUTIFY ? true : false,
-    preamble    : ARGS.preamble || null,
-    quote_style : ARGS.quotes != null ? ARGS.quotes : 0
+    beautify          : BEAUTIFY ? true : false,
+    preamble          : ARGS.preamble || null,
+    quote_style       : ARGS.quotes != null ? ARGS.quotes : 0,
```

Comment:
oh I initially added keep_quoted_props here, but then decided to move it to a single if statement. fixing